dockerfiles/init/manifests/codenvy.env

```diff
@@ -622,3 +627,16 @@
 #CODENVY_JMX_USERNAME=admin
 #CODENVY_JMX_PASSWORD=codenvy
 
+
+########################################################################################
+#####                                                                              #####
+#####                                     SOCAT                                    #####
+#####                                                                              #####
+# Expose socat port
+#     Expose socat port tobe available on host as 23750. It is needed only in all-in-one
+#     (basic mode), this will allow start a workspace without adding remote nodes for
+#     running worksoaces (production) mode.
+#     So if you are using remote nodes for running workspaces (production mode) this expose
+#     could be disabled.
+#     Enabled by default, to disable please uncomment and set CODENVY_EXPOSE_SWARM_PORT=false
+#CODENVY_EXPOSE_SWARM_PORT=true
```

Comment:
name is CODENVY_EXPOSE_SWARM_PORT=true
while it should be _SOCAT_ (no SWARM here)